docs/data/material/getting-started/templates/templates.md

```diff
@@ -40,9 +44,13 @@ You can find complete templates and themes like those shown below in the <a href
 
 ## Toolpad Core (beta)
 
-[Toolpad Core](https://mui.com/toolpad/core/introduction/) is a set of components designed for assembling dashboards and internal tools more efficiently.
-It builds on top of Material UI to provide sophisticated UI components with full-stack functionality, such as the [Dashboard Layout](https://mui.com/toolpad/core/react-dashboard-layout/) shown below.
-
-This is a great option if you don't want to start building from scratch but don't need a prebuilt template.
+If you're looking to build internal tools and dashboards fast, but don't want to start building from scratch, you can find examples of functional dashboards – with authentication, routing, and theming already integrated – in the [featured examples](http://mui.com/toolpad/core/introduction/examples/#featured-examples) section of the Toolpad Core docs.
 
-{{"demo": "../../components/app-bar/DashboardLayoutBasic.js", "height": 400, "iframe": true, "defaultExpanded": false, "bg": "inline"}}
+<a href="https://mui.com/toolpad/core/introduction/examples/#featured-examples">
+<span class="only-light-mode">
+<img src="/static/images/toolpad-core-template-light.png" alt="The Toolpad Core template comes with authentication, navigation and theming integrated" width="2280" height="1200" style="display: block;" />
+</span>
+<span class="only-dark-mode">
+<img src="/static/images/toolpad-core-template-dark.png" alt="The Toolpad Core template comes with authentication, navigation and theming integrated" width="2280" height="1200" style="display: block;"/>
```

Comment:
We have to set this as loading lazy, right now this starves other assets that are in the critical rendering path https://www.webpagetest.org/result/250102_BiDcKD_4G/1/details/#waterfall_view_step1

<img width="672" alt="SCR-20250102-ceok" src="https://github.com/user-attachments/assets/39d1c600-5207-40f9-9726-34d5897fde0e" />